spec/index.bs

```diff
@@ -2179,12 +2211,83 @@ interface GPUTextureUsage {
 
             **Arguments:**
             <pre class=argumentdef for="GPUDevice/createTexture(descriptor)">
-                descriptor: Description of the {{GPUTexture}} to create.
+                |descriptor|: Description of the {{GPUTexture}} to create.
             </pre>
 
             **Returns:** {{GPUTexture}}
 
-            Issue: Describe {{GPUDevice/createTexture()}} algorithm steps.
+            1. Issue the following steps on the [=Device timeline=] of |this|:
+                <div class=device-timeline>
+                    1. If any of the following requirements are unmet:
+                        <div class=validusage>
+                            - |this| must be a [=valid=] {{GPUDevice}}.
+                            - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/width=],
+                                |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/height=],
+                                and |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/depthOrArrayLayers=] must be greater than zero.
+                            - |descriptor|.{{GPUTextureDescriptor/mipLevelCount}} must be greater than zero.
+                            - |descriptor|.{{GPUTextureDescriptor/sampleCount}} must be either 1 or 4.
+                            - If |descriptor|.{{GPUTextureDescriptor/dimension}} is:
+                                <dl class="switch">
+                                    : {{GPUTextureDimension/"1d"}}
+                                    :: |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/width=] must be less than or equal to
+                                        |this|.{{GPUAdapterLimits/maxTextureDimension1D}}.
+                                    :: |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/height=] must be 1.
```

Comment:
I think it would structurally make more sense to do the following, though it doesn't make a big difference in the formatting:

```suggestion
                                    ::
                                        - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/width=] must be less than or equal to
                                            |this|.{{GPUAdapterLimits/maxTextureDimension1D}}.
                                        - |descriptor|.{{GPUTextureDescriptor/size}}.[=Extent3D/height=] must be 1.
```
and so on.